diff --git a/py/setup.py b/py/setup.py
index 3e7b6b83bb..40b7cb23e3 100755
--- a/py/setup.py
+++ b/py/setup.py
@@ -19,7 +19,6 @@ from distutils.command.install import INSTALL_SCHEMES
 from os.path import dirname, join, abspath
 from setuptools import setup
 from setuptools.command.install import install
-from setuptools_rust import Binding, RustExtension


 for scheme in INSTALL_SCHEMES.values():
@@ -84,12 +83,6 @@ setup_args = {
         "typing_extensions~=4.9",
         "websocket-client~=1.8",
     ],
-    'rust_extensions': [
-        RustExtension(
-            {"selenium-manager": "selenium.webdriver.common.selenium-manager"},
-            binding=Binding.Exec
-        )
-    ],
     'zip_safe': False
 }
